packages/datetime/src/dateRangeInput.tsx

```diff
@@ -712,6 +663,13 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
         return moment(dateString, this.props.format);
     }
 
+    private getHtmlPropsFromInputProps = (inputProps: HTMLInputProps & IInputGroupProps) => {
```

Comment:
Ah, true. Originally made this function when I still had two invocations (prior to introducing `renderInputGroup`).